packaging/ubuntu-14.04/snapd.postrm

```diff
@@ -82,6 +82,8 @@ if [ "$1" = "purge" ]; then
         umount -l "$mnt" || true
     done
     umount -l /run/snapd/ns/ || true
+    # remove all the current mount profiles that may exist.
+    rm -f /run/snapd/ns/snap.*.fstab
```

Comment:
Actually, you are correct. As soon as we unmount the *.mnt file snap-confine will behave correctly (it overwrites the .fstab file when it creates the namespace). I was worried about incorrect behaviour on a system where you purge snapd and re-install it without rebooting but I didn't realize this is already correct as-is.